05_syntax_lab/02.py

```diff
@@ -1,6 +1,22 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 1
+
+"""
+from random import randint
+
+random=randint(1, 100)

```

Comment:
Note unlike the previous exercise- here there's no reason to calculate the first value outside the loop. Also since loop index is not important here you can use the more concise:

```
sum = 0
for i in range(7): sum += randint(1,100)
```